__tests__/therapist.ts

```diff
@@ -0,0 +1,25 @@
+
+import request from 'supertest';
+import app from '.././src/app';
+import buildDB from "../src/db/build";
+import dbConnection from "../src/db/connection";
+

```

Comment:
why to delete index file, 
Instead we can tun `beforeAll` and `afterAll` at index and we can export all test suites at index as follows
```js
describe('therapist tests', therapistTest)
```